### DISCUSSION TITLE
运行和编译没问题，但是CS文件提示CS0103 InitializeComponent

### DISCUSSION THREAD

**Discussion by YSW33** [irihitech/Ursa.Avalonia#168] [Q&A] [answered]
![image](https://github.com/irihitech/Ursa.Avalonia/assets/141573398/604e7636-f4cd-4587-984a-01cbe6c9477c)

**Comment by YSW33** [comment]
具体引用了这些包
![image](https://github.com/irihitech/Ursa.Avalonia/assets/141573398/3fd9b542-2ee5-42db-91f5-51785d3af060)

**Comment by rabbitism** [comment]
应该只是source generator生成的代码没有被vs正确识别的问题

**Reply by YSW33** [reply]
我看AVALONIA那边似乎2022年就有这个问题，但是一直还在出现，是否是因为我最近升级VS导致？
![image](https://github.com/irihitech/Ursa.Avalonia/assets/141573398/e1040647-d0f6-4955-b39e-f5344812436a)

**Reply by rabbitism** [reply, marked as answer]
通常删除obj和bin文件夹再重新打开项目可以解决。

**Reply by YSW33** [reply]
删除obj和bin不行。

**Reply by YSW33** [reply]
avalonia回滚到11.0.7就没问题了。
